src/PluginClient.php

```diff
@@ -87,6 +117,17 @@ private function createPluginChain($pluginList)
             $firstCallable = $lastCallable;
         }
 
+        $firstCalls = 0;
+        $firstCallable = function (RequestInterface $request) use ($options, $lastCallable, &$firstCalls) {
```

Comment:
do we need the local $options? i thought since php 5.4 you can use $this inside the anonymous function.